### PR TITLE
Tune makefile

### PR DESCRIPTION
**Pull request by bdsaunders**
8 unused (unmentioned in Makefile so uninstalled) old files removed to attic.

### CODE REVIEW

File: linbox/matrix/sparsematrix/Makefile.am [hunk 1]
```diff
@@ -38,15 +40,20 @@ pkgincludesub_HEADERS =         \
...
-        sparse-parallel-vector.h         \
-        sparse-parallel-vector.inl       \
-        sparse-sequence-vector.h         \
+	sparse-parallel-vector.h         \
+	sparse-parallel-vector.inl       \
+	sparse-sequence-vector.h         \
 	triples-coord.h  \
 	read-write-sparse.h \
 	read-write-sparse.inl \
+<<<<<<< HEAD
```
Comment (ClementPernet) [inline linbox/matrix/sparsematrix/Makefile.am]: You have commited merge data (<<<HEAD, ====, etc). Please remove them.

Comment (bdsaunders) [reply linbox/matrix/sparsematrix/Makefile.am]: Oops.  Now fixed.  Thnx.

File: linbox/ring/gcd32.h [hunk 2]
```diff
@@ -1,195 +0,0 @@
-/* Copyright (C) LinBox
```
Comment (ClementPernet) [inline linbox/ring/gcd32.h]: I am not sure we want to remove gcd32.h and gcd64.h: they are used in `ring/poweroftwomodular.h` which has not been deleted in this PR.
Also it seems beyond a cleanup of Makefiles and would probably break users code based on poweroftwomodular.h

**Comment by bdsaunders** [comment]
OK.  I've restored these files.

I didn't find any usage of poweroftwomodular.h in linbox.  Just to have a
reference to it, I created a test for it.  The test fails as
poweroftwomodular.h includes a nonexistent file.  I put the test in the
skipped test list.

Poweroftwomodular looks like an incomplete project.

**Pull request merged**